rdf2vec/converters.py

```diff
@@ -2,6 +2,16 @@
 from tqdm import tqdm
 
 def create_kg(triples, label_predicates):
+    """Creates a knowledge graph according to triples and predicates label.
+
+    Args:
+        triples (list): The triples.
```

Comment:
Perhaps we could quickly state that each element in this list should be an iterable (tuple, list, ...) of 3 elements (s, p, o)